ci/test.yml

```diff
@@ -8,6 +8,9 @@ core unit test:
     - core unix regular build
   script:
     - nix-shell --run "poetry run make -C core test | ts -s"
+    # we need to generate qstrings for rust unit tests to work
+    - nix-shell --run "poetry run make -C core build_unix | ts -s"
```

Comment:
it would be better to add the required files to the artifact

Reply:
Fixed in 4aa1201.